pkg/stub/handler.go

```diff
@@ -406,10 +406,6 @@ func (h *Handler) CreateDefaultResourceIfNeeded(cfg *v1.Config) (*v1.Config, err
 		cfg.APIVersion = v1.GroupName + "/" + v1.Version
 		cfg = h.updateCfgArch(cfg)
 		switch {
-		// TODO as we gain content for non x86 platforms we can remove the nonx86 check
-		case util.IsNonX86Arch(cfg):
```

Comment:
Any reason not to remove the now-unused `IsNonX86Arch` function from `pkg/util/util.go` like in my PR?

Reply:
simple just in case / help refresh my memory / etc. ... understanding I may be the exception, I tend to not remove unused / commented out code immediately, but instead will go back periodically and do so.